packages/codemirror/src/extension.ts

```diff
@@ -776,6 +777,15 @@ export namespace EditorExtensionRegistry {
           title: trans.__('Line Wrap')
         }
       }),
+      Object.freeze({
+        name: 'dropCursor',
+        default: true,
+        factory: () => createConditionalExtension(dropCursor()),
+        schema: {
+          type: 'boolean',
+          title: trans.__('Drop Cursor')
```

Comment:
Backporting this will (should) warn about new translatable string.

I think it is a fine idea to have opt-out in case if this causes issues, though I think it would be also fine to always include. What you all think?

Reply:
> By opt-out would that meant not including the dropCursor extension if the string is localized in the given language?

No, I meant opt-out as having this as a setting in general, regardless of the localization. I was suggesting that not having a setting would be fine too.